src/main/java/io/github/lottetreg/httpserver/Redirect.java

```diff
@@ -0,0 +1,21 @@
+package io.github.lottetreg.httpserver;
+
+public class Redirect extends BaseRoute {
+  private String redirectPath;
+
+  Redirect(String path, String method, String redirectPath) {
+    super(path, method);
+    this.redirectPath = redirectPath;
+  }
+
+  public HTTPResponse getResponse(HTTPRequest request) {
+    String URI = "http://" + request.getHeader("Host") + getRedirectPath();
```

Comment:
Minor thing, don't address it now: you might want to abstract over "protocol" (`http://` vs `https://` vs `file://` vs `psql://`) at some point.